tests/reload/14-waiting/suite.rc

```diff
@@ -20,7 +25,9 @@ done
         script = true
     [[reloader]]
         script = """
+sleep 5
```

Comment:
Why are we waiting, this slee-ep is frustrating, why are we waiting, oh, why, why, why.

Reply:
Generally I have removed a whole bunch of these on this brunch, not sure why this one crept back in - I'll check...

Reply:
Removing...